nix/package.nix

```diff
@@ -12,20 +14,27 @@ stdenv.mkDerivation rec {
   src = ./..;
 
   nativeBuildInputs = [ zig scdoc pkg-config ];
-
-  buildInputs = [
-    libxml2
-  ];
+  buildInputs = [ libxml2 ];
 
   dontConfigure = true;
 
   preBuild = ''
     export HOME=$TMPDIR
+    cp -r ${zig-libxml2-src} ./vendor/zig-libxml2
   '';
 
   installPhase = ''
     runHook preInstall
     zig build -Drelease-safe -Dman-pages --prefix $out install
     runHook postInstall
   '';
+
+  outputs = [ "out" "dev" "man" ];
```

Comment:
Splits the package into:

`out` - the `.a` and `.so` files
`dev` - pkg-config stuff
`man` - man pages

out is default. to get e.g. man pages you run `nix build .#libflightplan.man` and you'll get a `result-man` dir

It just means you can drop man pages and other stuff if you dont need them